drakrun/drakrun/lib/sample_startup.py

```diff
@@ -64,23 +91,25 @@ def get_dll_startup_command(pe_data: bytes) -> str:
             for e in pe.DIRECTORY_ENTRY_EXPORT.symbols
         ]
     except AttributeError:
-        return "regsvr32 /s %f"
+        return "DllMain"
 
     for export in exports:
         if export[1] == "DllRegisterServer":
-            return "regsvr32 /s %f"
+            return "DllRegisterServer"
 
         if "DllMain" in export[1]:
-            return "rundll32 %f,{}".format(export[1])
+            return export[1]
 
     if exports:
         export = exports[0]
         if exports[0][1]:
-            return "rundll32 %f,{}".format(export[1].split("@")[0])
+            entrypoint = export[1].split("@")[0]
+            return entrypoint
         elif exports[0][0]:
-            return "rundll32 %f,#{}".format(export[0])
+            entrypoint = "#" + str(export[0])
+            return entrypoint
 
-    return "regsvr32 /s %f"
+    return "DllMain"
```

Comment:
previously the default was regsvr32 with DllRegisterServer, right?

Reply:
Same thing as above